Source/SwiftLintBuiltInRules/Rules/Lint/FunctionArgumentsSpacingRule.swift

```diff
@@ -0,0 +1,93 @@
+import SwiftSyntax
+
+@SwiftSyntaxRule
+struct FunctionArgumentsSpacingRule: Rule {
+    var configuration = SeverityConfiguration<Self>(.warning)
+
+    static let description = RuleDescription(
+        identifier: "functions_arguments_spacing",
+        name: "Function Arguments Spacing",
+        description: "Remove the space before the first function argument and after the last argument",
+        kind: .lint,
+        nonTriggeringExamples: [
+            Example("makeGenerator()"),
+            Example("makeGenerator(style)"),
+            Example("makeGenerator(true, false)"),
+            Example("""
+            makeGenerator(
+                a: true,
+                b: false
+            )
+            """)
+        ],
+        triggeringExamples: [
+            Example("makeGenerator(↓ )"),
```

Comment:
You may choose a shorter name for the function like just `f`. `makeGenerator` seems to imply a special meaning which it hasn't.